UI/window-basic-status-bar.cpp

```diff
@@ -127,15 +164,49 @@ void OBSBasicStatusBar::UpdateBandwidth()
 
 	double kbitsPerSec = double(bitsBetween) / timePassed / 1000.0;
 
+	if (kbitsPerSec == 0)
+		noBitrate = true;
+	else
+		noBitrate = false;
+
 	QString text;
 	text += QString("kb/s: ") +
 		QString::number(kbitsPerSec, 'f', 0);
+
 	kbps->setText(text);
 	kbps->setMinimumWidth(kbps->width());
 
 	lastBytesSent        = bytesSent;
 	lastBytesSentTime    = bytesSentTime;
 	bitrateUpdateSeconds = 0;
+
+	if (noBitrate) {
+		QPixmap pixmap(16, 16);
+		pixmap.fill(QColor("transparent"));
+		QPainter painter(&pixmap);
+		painter.setPen(Qt::NoPen);
+		painter.setBrush(QBrush(QColor(255,0,0)));
+		painter.drawRect(0, 0, 16, 16);
+		statusSquare->setPixmap(pixmap);
+	}
```

Comment:
Style:  Curly braces should be on the same line as non-function statement blocks (if, else if, else, switch, for, while, do).  See Line 191-192 and 200-201.